### PR TITLE
i18n(es): Updating error-reference.mdx

### PR DESCRIPTION
**Pull request by helmerdavila**
#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/es/reference/error-reference.mdx [hunk 1]
```diff
@@ -1,9 +1,4 @@
 ---
-# NOTE: This file is auto-generated from 'scripts/error-docgen.mjs'
-# Do not make edits to it directly, they will be overwritten.
-# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
-# Translators, please remove this note and the <DontEditWarning/> component.
-
```
Comment (helmerdavila) [inline src/content/docs/es/reference/error-reference.mdx:6]: Note says that I should remove this

File: src/content/docs/es/reference/error-reference.mdx [hunk 2]
```diff
@@ -68,7 +63,9 @@ La siguiente referencia es una lista completa de los errores que puedes encontra
 - [**FailedToFindPageMapSSR**](/es/reference/errors/failed-to-find-page-map-ssr/)<br/>Astro couldn't find the correct page to render
 - [**MissingLocale**](/es/reference/errors/missing-locale/)<br/>The provided locale does not exist.
 - [**MissingIndexForInternationalization**](/es/reference/errors/missing-index-for-internationalization/)<br/>Index page not found.
+- [**IncorrectStrategyForI18n**](/es/reference/errors/incorrect-strategy-for-i18n/)<br/> You can't use the current function with the current strategy
 - [**NoPrerenderedRoutesWithDomains**](/es/reference/errors/no-prerendered-routes-with-domains/)<br/>Prerendered routes aren't supported when internationalization domains are enabled.
+- [**MissingMiddlewareForInternationalization**](/es/reference/errors/missing-middleware-for-internationalization/)<br/>Enabled manual internationalization routing without having a middleware.
```
Comment (helmerdavila) [inline src/content/docs/es/reference/error-reference.mdx:68]: This page doesn't exist in spanish. Should I leave the link as it is?

File: src/content/docs/es/reference/error-reference.mdx (hunk 1 above)
Comment (dreyfus92) [reply src/content/docs/es/reference/error-reference.mdx:6]: Si la puedes quitar.

File: src/content/docs/es/reference/error-reference.mdx (hunk 2 above)
Comment (dreyfus92) [reply src/content/docs/es/reference/error-reference.mdx:68]: Hay que dejarlo por que falta traducir el error, lo marca el tracker.

**Review by dreyfus92 — APPROVED** [review]
Muchas gracias @helmerdavila, LGTM ✅

**Pull request merged**